gatsby-config.js

```diff
@@ -45,12 +45,12 @@ module.exports = {
       },
     },
     {
-      resolve: 'gatsby-plugin-fathom',
+      resolve: 'gatsby-plugin-manifest',
```

Comment:
does this plugin need to be in package.json as well?

Reply:
It's shipped with the theme but it's a good idea to add it explicitly, I'll add it.